src/config/custom.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 module.exports = {
-  timeout: 80 * 1000
+  timeout: 40 * 1000
```

Comment:
What about 5 secs and any test that needs more either should get it running below 5 secs or explicitly signal that? (and I say 5 and not 2 just because of CI vs user machines)